Maternal linkage + re-organization for consistency

CarolinePotteiger opened the pull request:
I tried to organize the commits so similar changes were grouped together. It looks like a lot, but is mostly moving files around so information is represented consistently in the repository. Please let me know of any questions!

CarolinePotteiger committed:
Implemented consistent representation of codesystems

Implemented consistent representation of codesystems

CarolinePotteiger committed:
consistent representation of bundles instances

consistent representation of bundles instances

CarolinePotteiger committed:
consistent representation of outcome bundles

CarolinePotteiger committed:
Removed files no longer in use

Removed files no longer in use

CarolinePotteiger committed:
Maternal linkage updates

- renamed value for plural and fetal death
- consistent language for URI value descriptions
- replaced maternallinkageupdateheader/maternallinkagesubmissionheader with updateheader/submissionheader
- re-organized files to be consistent with rest of the IG
- fixed examples

CarolinePotteiger committed:
Re-organize files and folders

Re-organized files and folders to be consistent across the IG and easier to find

CarolinePotteiger committed:
Update ignorewarnings for qa page and config file

Update ignorewarnings for qa page and config file

CarolinePotteiger committed:
Update narrative

Update narrative to include birth/fetal death use case and maternal linkage. Also fixed typos and errors

cabush commented on README.md:
```diff
@@ -1,22 +1,22 @@
-Vital records jurisdictions submit information on deaths in their jurisdiction to the U.S. National Center for Health Statistics (NCHS). For each submission, NCHS codes all causes of death, races, and ethnicities and returns the information to the submitter.  Similarly, jurisdictions submit information on births and fetal deaths to NCHS, and NCHS codes causes of fetal death, races and ethnicities ...
+Vital records jurisdictions submit information on deaths and births in their jurisdiction to the U.S. National Center for Health Statistics (NCHS). For each submission, NCHS codes all causes of death, races, and ethnicities and returns the information to the submitter.
 
-The [Vital Records Death Reporting (VRDR)](http://hl7.org/fhir/us/vrdr/) and [Birth and Fetal Death Reporting (BFDR)](http://hl7.org/fhir/us/fhir-bfdr/) FHIR IGs specifies how to represent the information sent from vital records jurisdictions to NCHS using FHIR documents. It does not specify how such data is exchanged nor how the coding information is represented and returned to the submitter.
```
I believe the initial description was correct in referring to "They do not..." As I am reading it is referring to both VRDR and BFDR, not just one IG.

cabush commented on README.md:
```diff
@@ -1,22 +1,22 @@
-The [Vital Records Death Reporting (VRDR)](http://hl7.org/fhir/us/vrdr/) and [Birth and Fetal Death Reporting (BFDR)](http://hl7.org/fhir/us/fhir-bfdr/) FHIR IGs specifies how to represent the information sent from vital records jurisdictions to NCHS using FHIR documents. It does not specify how such data is exchanged nor how the coding information is represented and returned to the submitter.
+The [Vital Records Death Reporting (VRDR)](https://hl7.org/fhir/us/vrdr/) and [Birth and Fetal Death (BFDR)](https://hl7.org/fhir/us/bfdr/) specify how to represent the information sent from vital records jurisdictions to NCHS using FHIR documents. They do not specify how such data is exchanged nor how the coding information is represented and returned to the submitter.  
 
-[FHIR Messaging](http://hl7.org/fhir/messaging.html) defines how to use FHIR in a message exchange scenario. This document describes the use of FHIR Messaging for:
```
Is the change dropping the 's' in http? When I click on the link I do see the url as https. Not sure if this matters or not.

cabush commented on input/fsh/profiles/messages/SD_MaternalLinkageSubmissionMessage.fsh:
Is the change from MaternalLinkageSubmissionHeader to SubmissionHeader meant to simplify the name referenced in the FSH code? Just curious on the change.

cabush commented on input/ignoreWarnings.txt:
```diff
@@ -38,230 +29,42 @@ INFORMATION: Bundle/ParentalDemographicsCodingMessage-Example1: Bundle: Can't fi
-WARNING: Bundle/IndustryOccupationCodingMessage-Example1: Bundle.entry[0].resource/*MessageHeader/IndustryOccupationCodingHeader-Example1*/.event.ofType(uri): URL value 'http://nchs.cdc.gov/vrdr_industryoccupation_coding' does not resolve
-WARNING: Bundle/IndustryOccupationCodingUpdateMessage1: Bundle.entry[0].resource.event.ofType(uri): URL value 'http://nchs.cdc.gov/vrdr_industryoccupation_coding_update' does not resolve
-WARNING: Bundle/IndustryOccupationCodingUpdateMessage1: Bundle.entry[0].resource/*MessageHeader/IndustryOccupationCodingUpdateHeader-Example1*/.event.ofType(uri): URL value 'http://nchs.cdc.gov/vrdr_industryoccupation_coding_update' does not resolve
+# Not necessary for this IG
```
Where these warnings removed or moved elsewhere?

cabush commented on input/ignoreWarnings.txt:
Were many elements moved or deleted? How is the ignoreWarnings.txt files used in FSH?

cabush commented on input/ignoreWarnings.txt:
Were components just moved? How is the ignoreWarnings.txt files used?

cabush commented on input/pagecontent/ImplementationNotes.md:
```diff
@@ -2,30 +2,30 @@
 * [NCHS Vital Records FHIR Messaging v1.0.1](https://nightingaleproject.github.io/vital_records_fhir_messaging_ig/v1.0.1/index.html)
 
-Periodically, NCHS will transition the submission process to use new published versions of these IGs.
+Periodically, NCHS will transition the submission process to use new published versions of these IGs and the [Birth and Fetal Death (BFDR) IG](https://hl7.org/fhir/us/bfdr/index.html).
```
Is this where BFDR IG will be added? For consistency, could we list the IG with the other two in the bulleted list?

cabush commented on input/pagecontent/StructureDefinition-VRM-CauseOfDeathCodingUpdateMessage-intro.md:
```diff
@@ -1,3 +1,3 @@
 
-The CodingMessage’s MessageHeader.response.identifier must equal the value of the MessageHeader.id property of the message that was coded. When processing coding responses this identifier must be used to associate the coding content with the message that was coded. This association ensures clarity when a death record is submitted or updated multiple times, and coding responses are delivered for a ...
+The CodingMessage’s MessageHeader.response.identifier must equal the value of the MessageHeader.id property of the message that was coded. When processing coding responses this identifier must be used to associate the coding content with the message that was coded. This association ensures clarity when a death, fetal death, or birth record is submitted or updated multiple times, and coding respon ...
```
If this is specific to coded cause of death then we don't need to list birth here.

cabush commented on input/pagecontent/StructureDefinition-VRM-ExtractionErrorMessage-intro.md:
```diff
@@ -2,5 +2,5 @@ The ErrorMessage’s MessageHeader.response.identifier must equal the value of t
 This association ensures clarity when a death record is submitted or updated multiple times, and error responses are delivered for a subset of these submissions.
 
-For submissions to NCHS, the set of current error messages are listed and described [here](business_rules.html).
+For submissions to NCHS, the set of current error messages are listed and described [here](business_rules_death.html).
```
Is there an update to the business rules and the reason for updating the link?

cabush commented on input/pagecontent/index.md:
```diff
@@ -83,11 +83,11 @@ NCHS and vital records jurisdictions need a mechanism to automatically recover f
 ### FHIR Messaging
 
-As described earlier, the [Vital Record Death Reporting (VRDR)STU2 FHIR IG](https://build.fhir.org/ig/HL7/vrdr/index.html) specifies how to represent the information sent from vital records jurisdictions to NCHS using FHIR documents. However, it does not specify the mechanism that is used to exchange those FHIR documents, nor how the coded response is represented and returned to the submitter. Th ...
+As described earlier, the [Vital Records Death Reporting (VRDR)](https://hl7.org/fhir/us/vrdr/) and [Birth and Fetal Death (BFDR)](https://hl7.org/fhir/us/bfdr/) specify how to represent the information sent from vital records jurisdictions to NCHS using FHIR documents. However, it does not specify the mechanism that is used to exchange those FHIR documents, nor how the coded response is represen ...
```
What would be the best approach in identifying the STU version that is used for certification and production with NCHS? I may be overlooking but is that documented within the VRFM? What comes to mind and helps me is the table Sarah provided and that is also in the transition document.

cabush commented on input/pagecontent/message.md:
As we are updating the narrative for birth and fetal death, I'm wondering if we should confirm that there are not any nuances we should document or consider. This might be a question for Sarah and/or Veronique to confirm. Unless you already have, please disregard.

cabush commented on sushi-config.yaml:
No direct comment but would like to go through the changes needed in the sushi-config for my edification

CarolinePotteiger committed:
Updated ignore warnings, implementation notes page, and corrected profile description

Updated ignore warnings, implementation notes page, and corrected profile description
